### PR TITLE
Upstream guides: Updates from Colleen

### PR DESCRIPTION
**Pull request by asettle**
Reference: https://etherpad.nue.suse.com/p/soc8-concerns

### CODE REVIEW

File: xml/user-glance.xml [hunk 1]
```diff
@@ -835,7 +835,7 @@ managed at a namespace level.
                     mapping in the following format:</para>
             <screen>{'images': [
   {'uri': 'http://glance.openstack.example.org/v1/images/71c675ab-d94f-49cd-a114-e12490b328d9',
-   'name': 'Ubuntu 10.04 Plain',
+   'name': 'SLE 15',
```
Comment (cmurphy) [inline xml/user-glance.xml]: Not positive but it might be better to call it SLES 15 or SUSE Linux Enterprise Server 15

Comment (cmurphy) [reply xml/user-glance.xml]: In our engineering cloud we call it SLES15

Comment (asettle) [reply xml/user-glance.xml]: Ah - I shouldn't have been so literal. Sorry.

Comment (asettle) [reply xml/user-glance.xml]: @dmpop do we have official style guide naming conventions we need to stick to here?

Comment (dmpop) [reply xml/user-glance.xml]: We have entities for both SLE and SUSE Linux Enterprise Server. But I'm not sure whether it makes sense to use them here. Just use SLES 15.

**Comment by asettle** [comment]
Note: https://github.com/SUSE-Cloud/doc-cloud/blob/ee1a3cccc9a566d75a2ad406c1f374c01d38289b/xml/admin-neutron-config-dhcp-ha.xml

**Pull request merged**